src/components/SubRequirement.vue

```diff
@@ -211,10 +353,10 @@ button.view {
   font-size: 14px;
   line-height: 14px;
   text-align: center;
-  color: white;
+  color: $white;
   text-transform: uppercase;
 }
-.completed-ptext span {
+.completed-ptext, .completed-ptext span {
```

Comment:
Are you sure about this change? See https://github.com/cornell-dti/course-plan/pull/197

and you are indeed causing a regression
<img width="384" alt="Screen Shot 2020-11-17 at 12 42 29" src="https://user-images.githubusercontent.com/4290500/99426654-5fc63300-28d2-11eb-9f4f-5a7dcedf2a00.png">

Reply:
I overlooked this case while trying to set the description of a completed sub-requirement to have the light gray and opacity: 0.8 text. I will undo this change and confirm with @einc how the descriptions of filled sub-requirements should look. Sorry about that!